src/components/Notification/StatusMessage.jsx

```diff
@@ -18,19 +21,28 @@ import React from 'react';
 const StatusMessage = ({ notification, locationUrl }) => {
   if (locationUrl) {
     return (
-      <dd className="labelStatus" role="alert">
-        {notification}{' '}
+      <Typography variant="dd">
+        {notification}
         <a href={locationUrl} target="_blank" rel="noreferrer">
-          {locationUrl}
+          Click here.
         </a>
-      </dd>
+      </Typography>
     );
   }
 
   return (
-    <dd className="labelStatus" role="alert">
+    <Typography
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center'
+      }}
+    >
       {notification}
-    </dd>
+      <br />
+      <CircularProgress />
```

Comment:
I think we can comment this out in the component and add a comment on top as a TODO.